### PR TITLE
feat(demo): Bare bones dev environment

### PR DESCRIPTION
**Pull request by Droogans**
Running `npm start` should automatically open a browser window to localhost:3000, and from there, `console.log` to [your browser's developer tools](https://developer.chrome.com/devtools) a `new Board().orbs`.

screen shot 2016-05-28 at 22 06 39

### CODE REVIEW

File: scripts/server.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+'use strict';
+
+const execSync = require('child_process').execSync;
```
Comment (rbcasperson) [inline scripts/server.js:4]: Ok I see this is built into Node.

File: scripts/server.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+let browserSync = require('browser-sync').create();
+
+execSync('npm run babel; npm run browserify', { stdio: [0, 1, 2] });
```
Comment (rbcasperson) [inline scripts/server.js:8]: The only thing I don't understand is what the `stdio: [0, 1, 2]` is.

Comment (Droogans) [inline scripts/server.js:8]: `execSync` will, by default, return a stream of output that you process into data by watching for new chunks, processing them, and joining them together into a giant blob of output. This output is not viewable by the user until everything is finished. No thanks.

By telling `execSync` to ignore the streaming output in our program, and instead pass it to the `standard input/output` (`stdio`) of the parent caller (in this case, a command line shell such as "Terminal"), we can both simplify our program, as well as hand off a stream to be immediately processed by the command line interface that called it. i.e., no waiting for it to finish. It just prints the output as it goes, just like you'd expect.

Comment (rbcasperson) [inline scripts/server.js:8]: Okay sounds simple and convenient enough, thanks!

**Pull request merged**